Clarify when monitors run after creation (#2867)
elastic/observability-docs: docs/en/observability/synthetics-get-started-project.asciidoc

[discrete]
= View in {kib}

[NOTE]
====

> When a monitor has been created (and also when edited), it will not run immediately but at some point after. After the first run occurs, the monitor will then run regularly based on the frequency configured for the monitor (e.g. every 10 minutes). You can run a manual test if you want to see the results more quickly.

Reviewer: What do you think of this? 

```suggestion
When a monitor is created or updated, it will not run immediately but at some point after. After the first run occurs, the monitor will begin running regularly based on the frequency configured for the monitor (for example, every 10 minutes). You can run a manual test if you want to see the results more quickly.
```

Also...

>at some point after

Can we be any more specific without giving a precise timeline? Maybe something like "typically less than a minute later" or "typically a few minutes later"?  (I don't actually know what is typical so feel free to adjust.)

Author: Thanks @colleenmcginnis I've applied your tweaks.

Regarding being more precise, we have a challenge there, in that the UI doesn't know (and we can't add this in before GA).

It will be less than the frequency, but on a 1 hour monitor, that could still in theory be 59 minutes away.

I wonder if we should just include an example to clarify the point?  For example:

> When adding a 10 minute monitor, you may have to wait up to 9 minutes for the first run, after which it will run every 10 minutes. When editing a monitor, you will need to wait for its next scheduled time for those changes to be reflected

Author: Thanks @colleenmcginnis I've gone with that copy in both locations
